addon/models/construction-project.js

```diff
@@ -155,7 +155,6 @@ export default class ConstructionProject extends XMLModel {
     {{/if}}
 
     {{#if model.isNew}}
-      {{! TODO}}
```

Comment:
Why did you remove the todo? We still have to find a way around this.

Reply:
Or can we remove the empty `<work>` in the new version?

Reply:
Nope, you're correct, I will return it.